python/cuml/test/test_preproc_utils.py

```diff
@@ -151,3 +151,12 @@ def assert_allclose(actual, desired, rtol=1e-05, atol=1e-05,
         assert diff_ratio <= ratio_tol
     else:
         return np_assert_allclose(actual, desired, rtol=rtol, atol=atol)
+
+
+def check_cupy8(func):
```

Comment:
We already have this function defined here https://github.com/rapidsai/cuml/blob/1ded565f6d2931242fd6072cb0787b1d9ac562f9/python/cuml/common/import_utils.py#L85

that version uses `LooseVersion` to be more robust, so I would recommend calling that function here

Reply:
@viclafargue I think this may be the only remaining issue?

Reply:
Done